website/src/components/Playground.vue

```diff
@@ -13,7 +14,7 @@ import { initializeParser, useAstGrep, Mode as ModeImport } from './astGrep'
 import '../style.css'
 
 // important initialization
-await initializeParser()
+// await initializeParser()
```

Comment:
this should not be removed

Reply:
Sorry, it's needed here. I commented it out during debugging and forgot to withdraw it.